core/store/src/db.rs

```diff
@@ -869,18 +890,18 @@ fn rocksdb_block_based_options(cache_size: usize) -> BlockBasedOptions {
 }
 
 // TODO(#5213) Use ByteSize package to represent sizes.
-fn choose_cache_size(col: DBCol) -> usize {
+fn choose_cache_size(col: DBCol, col_state_cache_size: usize) -> usize {
```

Comment:
consider either adding a config as a parameter, or moving this method into config